Client/spack.yaml

```diff
@@ -0,0 +1,8 @@
+spack:
+  specs:
+  - cmake@3.25.1
+  - spdlog@1.12.0
+  - mochi-thallium@0.10.1 ^libfabric@1.16.1 fabrics=mlx,rxd,rxm,shm,sockets,tcp,udp,verbs
+  - hdf5@1.14.1-2 +cxx +threadsafe
+  - cereal
```

Comment:
cereal is a dependency of thallium, no need to list explicitly, especially when no non-default configuration flags are needed.